packages/react-components/src/types/ChatMessage.ts

```diff
@@ -74,6 +74,34 @@ export interface ChatMessage extends MessageCommon {
    * {@link FileMetadata}
    */
   attachedFilesMetadata?: FileMetadata[];
+  /* @conditional-compile-remove(teams-inline-images) */
+  /**
+   * @beta
+   * A list of atachements message needs to make a request to download.
+   */
+
+  attachments?: ChatAttachment[];
+}
+/* @conditional-compile-remove(teams-inline-images) */
+/**
+ * @beta
+ * Status of a chat attachment.
+ */
+export type ChatAttachmentStatus = 'downloading' | 'completed' | 'failed' | 'none';
+
+/* @conditional-compile-remove(teams-inline-images) */
+/**
+ * A chat attachment object that holds data needed to download an attachment.
+ *
+ * @beta
+ */
+export interface ChatAttachment {
+  id: string;
+  attachmentType: string;
+  status: ChatAttachmentStatus;
+  name?: string;
+  url: string;
+  previewUrl?: string;
```

Comment:
Is previewUrl for inline attachment only? Or it is also for all file attachments

Reply:
Yes that is why it is optional, file attachments won't have a previewurl